lib/solid_queue/supervisor.rb

```diff
@@ -13,7 +13,7 @@ def start(**options)
         configuration = Configuration.new(**options)
 
         if configuration.valid?
-          new(configuration).tap(&:start)
+          SolidQueue::Supervisor::Launcher.new(configuration).tap(&:start)
```

Comment:
Could this be?
```suggestion
          Launcher.new(configuration).tap(&:start)
```
Just for consistency 😊